IMS/Infrastructure/Models/DTOs/LocationDTO.cs

```diff
@@ -0,0 +1,18 @@
+namespace IMS.Models
+{
+    public class LocationDTO
```

Comment:
Rename class to match .NET naming standards:

```suggestion
    public class LocationDto
```